vowpalwabbit/min_depth_tree.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
```

Comment:
Is there another consumer of this file?

Reply:
No.  Will collapse it into offset_tree.h